extendeddatacrossword_test.go

```diff
@@ -125,21 +125,24 @@ func BenchmarkRepair(b *testing.B) {
 			}
 
 			flattened := eds.flattened()
-			// Randomly remove 3/4 of the shares
-			for i := 0; i < i*i*3; {
-				ind := rand.Intn(i)
-				if len(flattened[ind]) == 0 {
-					continue
+			// Randomly remove 1/2 of the shares of each row
```

Comment:
I still think that we should also benchmark losing more data as it presents the worst-case scenario for repairing.

Reply:
The issue about removing more than 1/2 is that we're not guaranteed to be able to recover the whole square (i.e. it'll require more manual tweaks). The difference between recovering from 3/4 loss vs 1/2 loss should be a constant factor, so it _should_ irrelevant for benchmarking the relative changes optimizations bring.

Reply:
> so it should irrelevant for benchmarking 

Yeah, would be cool to validate this with a benchmark. But you are probably right and it's definitely orthogonal to this PR.